marimo/_runtime/validate_graph.py

```diff
@@ -57,8 +57,19 @@ def check_for_cycles(graph: DirectedGraph) -> dict[CellId_t, list[CycleError]]:
     errors = defaultdict(list)
     for cycle in graph.cycles:
         nodes_in_cycle: set[CellId_t] = set(sum(cycle, ()))
+        # before reporting the cells in the cycle to the user,
+        #   we first annotate the cycle with the variable names
+        #   that link its cells
```

Comment:
Super small nit on comment formatting.

```suggestion
        # before reporting the cells in the cycle to the user,
        # we first annotate the cycle with the variable names
        # that link its cells
```